cmd/influx/main.go

```diff
@@ -425,16 +425,17 @@ func (c *CommandLine) parseInto(stmt string) string {
 }
 
 func (c *CommandLine) Insert(stmt string) error {
-	stmt = stmt[7:]
-	if strings.EqualFold(stmt[:4], "into") {
-		stmt = c.parseInto(stmt[5:])
+	i, point := parseNextIdentifier(stmt)
+	if !strings.EqualFold(i, "insert") {
+		fmt.Println("ERR: missing space after INSERT")
```

Comment:
For the line `INSERTxyx`, it prints out `ERR: missing space after INSERT`, tosses the line, and displays a new prompt. Would you expect the CLI to attempt to write point `xyx`?